bookShop/src/app/modules/components/admin/order-component/order-list/order-list.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { LocalStorageService } from 'src/app/core/services/local-storage.service';
+import { IOrderData } from '../../../order/models/orderDataModel';
+
+@Component({
+  selector: 'app-order-list',
+  templateUrl: './order-list.component.html',
+  styleUrls: ['./order-list.component.scss'],
+})
+export class OrderListComponent implements OnInit {
+  orderData!: any[];
+
+  constructor(private localStorageService: LocalStorageService) {
+    this.orderData = this.localStorageService.getItem('orderData');
```

Comment:
Use `constructor` for DI and `ngOnInit` for initialization.

Reply:
Fixed